spec/lib/mercury/sync_spec.rb

```diff
@@ -7,10 +7,10 @@
   let!(:source) { 'test-exchange1' }
   let!(:queue) { 'test-queue1' }
   describe '::publish' do
-    %w{with without}.each do |w|
+    %w(with without).each do |w|
```

Comment:
@SniderThanYou In fact, you can use just about any non-alphanumeric character

```
2.2.2 :030 > %w|foo bar|
 => ["foo", "bar"] 
2.2.2 :031 > %w-foo bar-
 => ["foo", "bar"]
```

This feature originated in `sed`, if I'm not mistaken.